### PR TITLE
Update redirected URLs (part 40) 

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updates redirected URLs.

### CODE REVIEW

File: files/en-us/web/api/rsahashedkeygenparams/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The **`RsaHashedKeyGenParams`** dictionary of the [Web Crypto API](/en-US/docs/W
 - `name`
   - : A {{domxref("DOMString")}}. This should be set to `RSASSA-PKCS1-v1_5`, `RSA-PSS`, or `RSA-OAEP`, depending on the algorithm you want to use.
 - `modulusLength`
-  - : A `Number`. The length in bits of the RSA modulus. This should be at least 2048: see for example see [NIST SP 800-131A Rev. 1](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-1/final). Some organizations are now recommending that it should be 4096.
+  - : A `Number`. The length in bits of the RSA modulus. This should be at least 2048: see for example see [NIST SP 800-131A Rev. 1](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-1/archive/2015-11-06). Some organizations are now recommending that it should be 4096.
```
Comment (OnkarRuikar) [inline files/en-us/web/api/rsahashedkeygenparams/index.md:20]:
```suggestion
  - : A `Number`. The length in bits of the RSA modulus. This should be at least 2048: see for example see [SP 800-131A Rev. 2](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-2/final). Some organizations are now recommending that it should be 4096.
```

File: files/en-us/web/api/rtcdatachannel/index.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ _Also inherits methods from {{DOMxRef("EventTarget")}}._
 
 ## Data format
 
-The underlying data format is defined by the IEEE draft specification [`draft-ietf-mmusic-sctp-sdp`](https://datatracker.ietf.org/doc/draft-ietf-mmusic-sctp-sdp). The current format specifies its protocol as either `"UDP/DTLS/SCTP"` (UDP carrying DTLS carrying SCTP) or `"TCP/DTLS/SCTP"` (TCP carrying DTLS carrying SCTP). Older browsers may only specify `"DTLS/SCTP"`.
+The underlying data format is defined by the IEEE draft specification [`draft-ietf-mmusic-sctp-sdp`](https://datatracker.ietf.org/doc/rfc8841/). The current format specifies its protocol as either `"UDP/DTLS/SCTP"` (UDP carrying DTLS carrying SCTP) or `"TCP/DTLS/SCTP"` (TCP carrying DTLS carrying SCTP). Older browsers may only specify `"DTLS/SCTP"`.
```
Comment (OnkarRuikar) [inline files/en-us/web/api/rtcdatachannel/index.md:121]:
```suggestion
The underlying data format is defined by the IEEE specification [SDP Offer/Answer Procedures for SCTP over DTLS Transport(RFC 8841)](https://datatracker.ietf.org/doc/rfc8841/). The current format specifies its protocol as either `"UDP/DTLS/SCTP"` (UDP carrying DTLS carrying SCTP) or `"TCP/DTLS/SCTP"` (TCP carrying DTLS carrying SCTP). Older browsers may only specify `"DTLS/SCTP"`.
```

File: files/en-us/web/api/subtlecrypto/generatekey/index.md [hunk 3]
```diff
@@ -151,4 +151,4 @@ let key = await window.crypto.subtle.generateKey(
 ## See also
 
 - [Cryptographic key length recommendations](https://www.keylength.com/).
-- [NIST cryptographic algorithm and key length recommendations](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-1/final).
+- [NIST cryptographic algorithm and key length recommendations](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-1/archive/2015-11-06).
```
Comment (OnkarRuikar) [inline files/en-us/web/api/subtlecrypto/generatekey/index.md:154]:
```suggestion
- [NIST Transitioning the Use of Cryptographic Algorithms and Key Lengths](https://csrc.nist.gov/publications/detail/sp/800-131a/rev-1/archive/2015-11-06).
```

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup/index.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ So let's get started by setting up the basis for our WebRTC-powered phone app.
 3. Next, install the required dependencies using the following commands:
 
     - [Express](https://expressjs.com/): `yarn add express`
-    - [PeerJS](https://peerjs.com/docs.html#api): `yarn add peerjs`
+    - [PeerJS](https://peerjs.com/docs/#api): `yarn add peerjs`
```
Comment (OnkarRuikar) [inline files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup/index.md:14]:
```suggestion
    - [PeerJS](https://peerjs.com/docs/): `yarn add peerjs`
```

File: files/en-us/web/css/flex-basis/index.md [hunk 5]
```diff
@@ -60,7 +60,7 @@ The `flex-basis` property is specified as either the keyword `content` or a `<'w
 
   - : Indicates automatic sizing, based on the flex item's content.
 
-    > **Note:** This value was not present in the initial release of Flexible Box Layout, and thus some older implementations will not support it. The equivalent effect can be had by using `auto` together with a main size ([width](https://drafts.csswg.org/css2/visudet.html#propdef-width) or [height](https://drafts.csswg.org/css2/visudet.html#propdef-height)) of `auto`.
+    > **Note:** This value was not present in the initial release of Flexible Box Layout, and thus some older implementations will not support it. The equivalent effect can be had by using `auto` together with a main size ([width](https://drafts.csswg.org/css2/#propdef-width) or [height](https://drafts.csswg.org/css2/#propdef-height)) of `auto`.
```
Comment (OnkarRuikar) [inline files/en-us/web/css/flex-basis/index.md:63]:
```suggestion
    > **Note:** This value was not present in the initial release of Flexible Box Layout, and thus some older implementations will not support it. The equivalent effect can be had by using `auto` together with a main size ([width](https://drafts.csswg.org/css2/#the-width-property) or [height](https://drafts.csswg.org/css2/#the-height-property)) of `auto`.
```

File: files/en-us/web/guide/ajax/wai_aria_live_regions_api_support/index.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ Firefox 3 contains important improvements to the way the Mozilla engine exposes
 
 These features will help screen reader developers improve the quality and performance of live region support, both for pages that are marked up with ARIA live region markup, and for pages where the author did not add any additional markup.
 
-Please read the [ARIA spec](https://www.w3.org/WAI/PF/aria/#liveregions) or the [live region report](/en-US/docs/Web/Guide/AJAX/WAI_ARIA_Live_Regions_API_Support) to learn about ARIA live region markup.
+Please read the [ARIA spec](https://www.w3.org/TR/wai-aria/#liveregions) or the [live region report](/en-US/docs/Web/Guide/AJAX/WAI_ARIA_Live_Regions_API_Support) to learn about ARIA live region markup.
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/ajax/wai_aria_live_regions_api_support/index.md:14]:
```suggestion
Please read the [ARIA spec](https://www.w3.org/TR/wai-aria/#dfn-live-region) or the [live region report](/en-US/docs/Web/Guide/AJAX/WAI_ARIA_Live_Regions_API_Support) to learn about ARIA live region markup.
```

File: files/en-us/web/api/element/setattribute/index.md [hunk 7]
```diff
@@ -83,7 +83,7 @@ This demonstrates two things:
 - The first call to `setAttribute()` above shows changing the `name` attribute's value to "helloButton".
   You can see this using your browser's page inspector ([Chrome](https://developer.chrome.com/docs/devtools/css/),
   [Edge](https://docs.microsoft.com/microsoft-edge/f12-devtools-guide/dom-explorer),
-  [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/index.html), [Safari](https://support.apple.com/guide/safari-developer/welcome/mac)).
+  [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/index.html), [Safari](https://support.apple.com/en-in/guide/safari-developer/welcome/mac)).
```
Comment (teoli2003) [inline files/en-us/web/api/element/setattribute/index.md:86]: Safari is guessing the locale. As we are in _en-US_ here, let's use this one.

```suggestion
  [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/index.html), [Safari](https://support.apple.com/en-us/guide/safari-developer/welcome/mac)).
```

File: files/en-us/web/api/web_audio_api/index.md [hunk 8]
```diff
@@ -115,7 +115,7 @@ Interfaces for defining effects that you want to apply to your audio sources.
 - {{domxref("PeriodicWave")}}
   - : Describes a periodic waveform that can be used to shape the output of an {{ domxref("OscillatorNode") }}.
 - {{domxref("IIRFilterNode")}}
-  - : Implements a general **[infinite impulse response](https://en.wikipedia.org/wiki/infinite%20impulse%20response "infinite impulse response")** (IIR) filter; this type of filter can be used to implement tone control devices and graphic equalizers as well.
+  - : Implements a general **[infinite impulse response](https://en.wikipedia.org/wiki/Infinite_impulse_response "infinite impulse response")** (IIR) filter; this type of filter can be used to implement tone control devices and graphic equalizers as well.
```
Comment (teoli2003) [inline files/en-us/web/api/web_audio_api/index.md:118]: Drive-by extra fixes, removal of the useless bold marks:
```suggestion
  - : Implements a general [infinite impulse response](https://en.wikipedia.org/wiki/Infinite_impulse_response "infinite impulse response") (IIR) filter; this type of filter can be used to implement tone-control devices and graphic equalizers as well.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A couple of changes.

**Review by teoli2003 — APPROVED** [review]
